DATA_LICENSE.txt

```diff
@@ -0,0 +1,22 @@
+Organic Maps Binary Data License
+
+The Organic Maps project provides compiled binary data files (including but not limited to `.mwm` map files) under a separate license from the source code.
```

Comment:
What are other binary files produced/distributed?
It might make sense to list them so that fork maintainers could check their compliance status easily.